src/appengine/handlers/testcase_detail/show.py

```diff
@@ -152,6 +160,16 @@ def highlight_common_stack_frames(crash_stacktrace):
   return '\n'.join(highlighted_crash_stacktrace_lines)
 
 
+def _unescape_android_kernel_link(line):
```

Comment:
This escaping and unescaping is very hacky, also, we shouldn't do html in stack_analyzer. in stack_analyzer, we can just just have
http://go/pakernel/msm-google/+/40e9b2ff3a280a8775cfcd5841e530ce78f94355/fs/open.c#622
or like
http://go/pakernel/msm-google/+/40e9b2ff3a280a8775cfcd5841e530ce78f94355/fs/open.c#622;fs/open.c:622 and then linkify seeing http:/ start and ; seperator or some other seperator.
and then we can have a function here called linkify_android_kernel_stack_frame_if_needed.
We also need to speed this function as this linkification is not needed for most usecases, so before start of loop (for line in crash_stacktrace.splitlines(:) we can do a bool to see if this is kasan stack and only do linkify if true.